ui/components/ui/new-network-info/new-network-info.js

```diff
@@ -60,11 +60,13 @@ export default function NewNetworkInfo() {
     }
   }, [providerConfig.chainId]);
 
+  /*
   useEffect(() => {
     checkTokenDetection();
     // we want to only fetch once
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, []);
+  */
```

Comment:
This is temporary, caused by another bug.

Reply:
This is fixed on develop now so you can uncomment this